data/explorer/beaconSetMetadata.json

```
@@ -50,5 +50,14 @@
     "logos": ["USDT", "USD"],
     "decimalPlaces": 4,
     "prefix": "$"
+  },
+  "0x8620ac32e01e44bd537eda39831f1c154fab786223356b30337553a2a63150af": {
+    "category": "Cryptocurrency",
+    "pricingCoverage": {
+      "polygon-testnet": "test-pricing-set-free"
+    },
+    "logos": ["STABLE", "USD"],
```

I put `"STABLE"` because I didn't know which logo to pick 😁

---

😅 so `logos` must reference [`commonLogos`](https://github.com/api3dao/operations/blob/82bded9cb99a4fc8d03f02892f689d8598f80fbf/data/explorer/commonLogos.json) (it's a mapping to a URL for a logo). The original idea was to represent beaconsets with logos for the underlying constituents (so three constituents, three logos). It may be good to test that, so perhaps
```suggestion
    "logos": ["BUSD", "USDT", "DAI", "USDC"],
```
This is actually a pretty cool dAPI...

---

There's a good chance that the market will ignore the extra logos as this is will be the first time we're using >2 logo functionality.

---

That makes a lot of sense 👏🏻  Maybe I should have checked the api3 market frontend code to know exactly what this array was used for 🤔